tapdb/postgres.go

```diff
@@ -72,6 +72,10 @@ func NewPostgresStore(cfg *PostgresConfig) (*PostgresStore, error) {
 		return nil, err
 	}
 
+	rawDb.SetMaxOpenConns(maxConns)
```

Comment:
Shouldn't this be `cfg.MaxOpenConnections` with a fall back to this constant if the config value is 0?

Reply:
Ah yeah, need to respect the config, will add that. I'm seeing this actively on the testnet universe server, and also could be the culprit of some of the issues we're seeing.